trieste/models/model_interfaces.py

```diff
@@ -419,6 +419,9 @@ def covariance_between_points(
 
         :return: Covariance matrix between the sets of query points with shape [N, M]
         """
+        if isinstance(self.model, SGPR):
```

Comment:
it would be good to raise an Issue on GIthub regarding the covariance for SGPRs, so we have it on our to do list - probably it would entail a subclass of `GaussianProcessRegression` where covariance method is additionally implemented

Reply:
Done! See #329